src/SFA.DAS.EmployerFinance.Web/Views/Transfers/Index.cshtml

```diff
@@ -48,7 +48,17 @@
             <a href="/accounts/@Model.Data.HashedAccountID/transfers/financial-breakdown" class="govuk-button govuk-button--secondary">View my transfer finances</a>
 
             <h2 class="govuk-heading-m">My pledges</h2>
-            <p>Create a public funding pledge which is shown online so that businesses can apply to you for a transfer of funds.</p>
+            @if (Model.Data.HasMinimumTransferFunds)
```

Comment:
I could be wrong, but I think this is the wrong way around? Don't we show this panel if they _don't_ have enough money, as it explains the disabled button to the user? Might have misread it all tho.

Reply:
You are correct. I will update.